src/common/api/types.ts

```diff
@@ -220,12 +226,6 @@ export interface Cluster {
    * {"networking":{"networkType": "OVNKubernetes"},"fips":true}
    */
   installConfigOverrides?: string;
-  /**
-   * Json formatted string containing the user overrides for the initial ignition config
-   * example:
-   * {"ignition": {"version": "3.1.0"}, "storage": {"files": [{"path": "/tmp/example", "contents": {"source": "data:text/plain;base64,aGVscGltdHJhcHBlZGluYXN3YWdnZXJzcGVj"}}]}}
-   */
-  ignitionConfigOverrides?: string;
```

Comment:
Same as `swagger.yaml`. We only update the parts relevant to the specific issue.